### DISCUSSION TITLE
How to use with CloudFlare images?

### DISCUSSION THREAD

**Discussion by onurusluca** [nuxt/image#1491] [Q&A]
Hi, I am trying to get NuxtImg going with CloudFlare images. https://image.nuxt.com/providers/cloudflare
It says on the docs about baseURL `Your deployment's domain (zone).` meaning I have to connect my website domain to CloudFlare, like zones?

I am using Vercel's automatically assigned domain name for now. It is not possible to change DNS afaik. So does it mean I can't test my website on local or just dev environment?

Or is it just `https://imagedelivery.net/<account_id>`? Even if I do this, for some reason, the image url becomes: /https://imagedeliver... It adds a '/'. 

Also on docs, it shows only the image name as src: `src="/burger.jpeg"`, does it mean this part: /<account_id>/<image_id>/public`?

Is there an example?

If possible, and if this gets resolved I want to open a PR for docs.

**Comment by ymansurozer** [comment]
This is how I was able to do this, hope it helps: https://github.com/nuxt/image/issues/805#issuecomment-2483628222
